<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainUI</class>
 <widget class="QMainWindow" name="MainUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>6</number>
      </property>
      <widget class="QWidget" name="page_welcome">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>80</y>
          <width>79</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Welcome</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_partitions">
       <widget class="QTreeWidget" name="treeWidget">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>150</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="page_user"/>
      <widget class="QWidget" name="page_pkgs">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTabWidget" name="tabWidget_2">
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>Tab 1</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QTreeWidget" name="treeWidget_2">
              <column>
               <property name="text">
                <string notr="true">1</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>Tab 2</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_summary"/>
      <widget class="QWidget" name="page_installing">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Tab 1</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Tab 2</string>
           </attribute>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QProgressBar" name="progressBar_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_installing">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Installing...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_finished"/>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_buttons">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QProgressBar" name="progress_pages">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>9</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
         <property name="format">
          <string>Page %v/%m</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="tool_prev">
         <property name="text">
          <string>Previous</string>
         </property>
         <property name="icon">
          <iconset theme="arrow-left">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="shortcut">
          <string>Alt+P</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tool_next">
         <property name="text">
          <string>Next</string>
         </property>
         <property name="icon">
          <iconset theme="arrow-right">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="shortcut">
          <string>Alt+N</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tool_startinstall">
         <property name="text">
          <string>Start Install</string>
         </property>
         <property name="icon">
          <iconset theme="dialog-ok-apply">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tool_reboot">
         <property name="text">
          <string>Reboot Now</string>
         </property>
         <property name="icon">
          <iconset theme="system-reboot">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionInfo"/>
   <addaction name="actionKeyboard"/>
   <addaction name="actionLocale"/>
  </widget>
  <action name="actionInfo">
   <property name="icon">
    <iconset theme="utilities-system-monitor">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>System</string>
   </property>
   <property name="toolTip">
    <string>System Information</string>
   </property>
  </action>
  <action name="actionKeyboard">
   <property name="icon">
    <iconset theme="input-keyboard">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Keyboard</string>
   </property>
   <property name="toolTip">
    <string>Keyboard Settings</string>
   </property>
  </action>
  <action name="actionLocale">
   <property name="icon">
    <iconset theme="preferences-desktop-locale">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Locale</string>
   </property>
   <property name="toolTip">
    <string>Localization</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
